template/src/ontology/Makefile.jinja2

```diff
@@ -634,7 +634,7 @@ pattern_schema_checks dosdp_validation: $(TMPDIR)/pattern_schema_checks
 
 .PHONY: update_patterns
 update_patterns: download_patterns
-	if [ -f $(TMPDIR)/dosdp/*.yaml ]; then cp -r $(TMPDIR)/dosdp/*.yaml $(PATTERNDIR)/dosdp-patterns; fi
+	if [ -d $(TMPDIR)/dosdp ]; then cp -r $(TMPDIR)/dosdp/*.yaml $(PATTERNDIR)/dosdp-patterns; fi
```

Comment:
This wont work:

```
$ ls tmp/dosdp/

$ cp -r tmp/dosdp/*.yaml .
cp: tmp/dosdp/*.yaml: No such file or directory
```

Reply:
Sorry, I don't understand. The issue is not to fail when `$(TMPDIR)/dosdp` doesn't exist. 
I tested it here in CL, and it works for this.

```bash
➜  ontology git:(master) ✗ sh run.sh make update_patterns
if [ true  = true ]; then rm -f tmp/dosdp/*.yaml.1 || true; fi
if [ true  = true ] && [ -s ../patterns/dosdp-patterns/external.txt ]; then wget -i ../patterns/dosdp-patterns/external.txt --backups=1 -P tmp/dosdp; fi
if [ true  = true ]; then rm -f tmp/dosdp/*.yaml.1 || true; fi
if [ -d tmp/dosdp ]; then cp -r tmp/dosdp/*.yaml ../patterns/dosdp-patterns; fi
➜  ontology git:(master) ✗ 
```

Reply:
Yes, but if you have tmp/dosdp/ directory, but no patterns in it the pipeline will still fail! (and it should not)..